htlcswitch/link.go

```diff
@@ -1114,6 +1126,10 @@ func (l *channelLink) htlcManager() {
 		case pkt := <-l.downstream:
 			l.handleDownstreamPkt(pkt)
 
+		case msg := <-l.localDownstream:
+			l.handleDownstreamPkt(msg.pkt)
```

Comment:
Should add basic unit test that covers this new code path.

Reply:
It is covered, for example in `TestChannelLinkSingleHopPayment`